experiments/training/read_prom.py

```diff
@@ -0,0 +1,61 @@
+import json
+import pandas
+import os, logging
+import time
+from datetime import timedelta, datetime
+from prometheus_api_client.utils import parse_datetime
+from prometheus_api_client import PrometheusConnect, MetricRangeDataFrame
+import urllib3
+from flask import Flask
+from jproperties import Properties
+urllib3.disable_warnings()
+
+configs = Properties()
+
+start_time = parse_datetime("1d")
+end_time = parse_datetime("now")
+chunk_size = timedelta(days=1)
+
+app = Flask(__name__)
+
+def loadProperties():
+    with open('metrics.properties', 'rb') as config_file:
+        configs.load(config_file)
+    metrics = configs.get("metrics").data
+    return metrics.split(',')
+metrics_list = loadProperties()
+
+
+if(os.getenv('PROMETHEUS') != None):
```

Comment:
I see we are trying to use ENV_VAR but no ENV_VARS has been set in the docker compose file.